toolboxes/modules/heatfluid/pages/theory.adoc

```diff
@@ -1,4 +1,7 @@
-= Heat Fluid
+= Theory of Heat Transfer and Fluid
```

Comment:
The title is not correct. `Heatfluid` or `Conjugate heat transfer`. Maybe for now heat fluid because the name of the toolbox